### ISSUE TITLE
WCAG 1.4.3: Elements must have sufficient color contrast (.hide-for-800-and-below)

### ISSUE THREAD

**Issue by zcjess** [zcjess/Spire#4]
#### Issue
`Elements must have sufficient color contrast` ([color-contrast](https://accessibilityinsights.io/info-examples/web/color-contrast))
#### Target application
[Spire](https://staging.spirelearn.com/programs/1/homepage)
#### Element path
.spire-button--secondary.btn-normal[href$="opportunities"] > .hide-for-800-and-below
#### Snippet
`<span class="hide-for-800-and-below">Explore</span>`

Fix the following:
Element has insufficient color contrast of 4.21 (foreground color: #0069d2, background color: #cfe8ff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1Element has insufficient color contrast of 4.21 (foreground color: #0069d2, background color: #cfe8ff, font size: 12.0pt (16px), font weight: normal). Expected contrast ratio of 4.5:1
Use foreground color: #0063c7 and the original background color: #cfe8ff to meet a contrast ratio of 4.61:1Use foreground color: #0063c7 and the original background color: #cfe8ff to meet a contrast ratio of 4.61:1
Or use background color: #d9f3ff and the original foreground color: #0069d2 to meet a contrast ratio of 4.61:1.
![image](https://github.com/zcjess/Spire/assets/90011962/a1a5cf97-8123-4c93-b1bc-3218ca2968b0)
![image](https://github.com/zcjess/Spire/assets/90011962/fc166d20-84f8-43a7-a08c-40afc4445e70)

#### Environment
Chrome version 114.0.0.0
This accessibility issue was found using Accessibility Insights for Web 2.38.1 (axe-core 4.6.3), a tool that helps find and fix accessibility issues. Get more information & download this tool at [http://aka.ms/AccessibilityInsights](http://aka.ms/AccessibilityInsights).

**Comment by zcjess** [comment]
Element has insufficient color contrast of 1.74 (foreground color: #c4c4c4, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1Element has insufficient color contrast of 1.74 (foreground color: #c4c4c4, background color: #ffffff, font size: 10.5pt (14px), font weight: normal). Expected contrast ratio of 4.5:1
Use foreground color: #727272 and the original background color: #ffffff to meet a contrast ratio of 4.81:1.
![image](https://github.com/zcjess/Spire/assets/90011962/ac40e5d2-b08e-42dc-987d-f68dac64cd4c)
